nums/models/glms.py

```diff
@@ -462,8 +490,21 @@ def irls(
     return beta
 
 
-def lbfgs():
-    raise NotImplementedError()
+# pylint: disable = unused-argument
+def lbfgs(
+    app: ArrayApplication,
+    model: GLM,
+    beta,
+    X: BlockArray,
+    y: BlockArray,
+    tol: BlockArray,
+    max_iter: int,
+):
+    # TODO (hme): Enable a way to provide memory length and line search parameters.
```

Comment:
Where exactly do we want to provide line search params. What kind of params would these be. would they be kind of line search, like backtracking, etc and max iters and other line search params like alpha etc..

Reply:
Yes, I think the best solution here is to allow a way to provide those params via the GLM class constructor `__init__`. For backtracking, you can see a number of parameters hardcoded in the LBFGS class, including initial alpha value, how much it is reduced per iteration, and the slope param.